ipaclient/plugins/vault.py

```diff
@@ -752,8 +751,7 @@ def forward(self, *args, **options):
                 error=_('Invalid vault type'))
 
         # initialize NSS database
-        current_dbdir = paths.IPA_NSSDB_DIR
```

Comment:
I rather think the author of the code copy and pasted some code and kept the style.